RELEASING.md

```diff
@@ -0,0 +1,20 @@
+# Release protocol
+This document describes the steps that need to be taken to create a new release for VAST.
+
+- Verify that the documentation in the `README.md`, the man page (`docs/man.1.md`) and https://github.com/vast-io/manual are up to date.
+- Create the pre tag commit:
+  - Check that the `CHANGELOG.md` is complete by going through the list and comparing to the output of this one-liner:
+    ``` sh
+    git log (git describe --tags --abbrev=0 origin/master)..origin/master --first-parent --pretty="- %h %s%w(0,0,4)%+b"
+    ```
+  - Add a `##` Header with the new tag version and date at the top of `CHANGELOG.md`:
+    ``` md
+    ## v0.1.2 (2018-11-11)
+    ```
+  - Set the version number in `VERSION` to the desired number. The version number should already be correct in most cases, so you only need to remove the `-pre` suffix.
+- Create the release on [vasts releases page](https://github.com/vast-io/vast/releases).
+- Create a post tag commit:
+  - Set the version number in `VERSION` to the next minor version increment and add the *`-pre`* suffix.
```

Comment:
I seems to me that the next version is a parameter to the release script: for example, after 0.7 we may decide to go to 1.0 instead of 0.8.

Reply:
This is just one way of dealing with version numbers. Increment before release is another. I would like to mark unreleased revisions somehow though, that's why I added the `-pre` stuff.

Reply:
Yeah, a suffix is a good idea. Have you thought about using `-beta`? This is how Zeek does it.